pom.xml

```diff
@@ -9,10 +9,10 @@
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>com.alexx</groupId>
-    <artifactId>employeesAndDepartments</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
-    <name>employeesAndDepartments</name>
-    <description>employeesAndDepartments</description>
+    <artifactId>employees_and_departments</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>employees_and_departments</name>
```

Comment:
Please read this: https://stackoverflow.com/questions/3724415/maven-artifact-and-groupid-naming
And then make changes